api/nodes/serializers.py

```diff
@@ -373,3 +379,29 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeInstitutionRelationshipSerializer(JSONAPIRelationshipSerializer):
+
+    class Meta:
+        type_ = 'institution'
+
+    def relationship(self, obj):
+        return obj.primary_institution
+
+    def update(self, instance, validated_data):
+        node = instance
+        user = self.context['request'].user
+        data = self.context['request'].data['data']
+        if data:
+            assert data['type'] == 'institution', 'Not right endpoint for this type, expected \'institution\' '
```

Comment:
This line can be removed if you decide to make type=TypeField(...) a field on the serializer.